src/components/portrait/portrait.pw.tsx

```diff
@@ -0,0 +1,486 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { getDataElementByValue } from "../../../playwright/components";
+import {
+  portraitImage,
+  portraitInitials,
+  portraitPreview,
+} from "../../../playwright/components/portrait/index";
+
+import { icon } from "../../../playwright/components/index";
+
+import {
+  CHARACTERS,
+  COLOR,
+  SIZE,
+  VALIDATION,
+} from "../../../playwright/support/constants";
+import { checkAccessibility } from "../../../playwright/support/helper";
+import Box from "../../../src/components/box";
+import {
+  PORTRAIT_SIZE_PARAMS,
+  PORTRAIT_SIZES,
+} from "../../../src/components/portrait/portrait.config";
+import {
+  PortraitComponent,
+  PortraitDefaultComponent,
+} from "./components.test-pw";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
```

Comment:
**nitpick** (non-blocking): ironically here I would rather see some blank lines separating the definitions of these consts. Really just personal opinion as I think it's easier on the eye - feel free to ignore if you disagree. (These blank lines were there in the original Cypress version - I think my automatic conversion script strips them out so you can blame me!)